plugins/source/test/resources/plugin/client.go

```diff
@@ -98,6 +90,15 @@ func Configure(_ context.Context, logger zerolog.Logger, spec []byte, opts plugi
 	if err := config.Validate(); err != nil {
 		return nil, fmt.Errorf("failed to validate spec: %w", err)
 	}
+
+	if opts.NoConnection {
```

Comment:
why did the `NoConnection` section move? I thought we shouldn't validate config if NoConnection is given

Reply:
The config doesn't have any required params so it's to accomodate the default values for the table config. Test plugin is doing its own thing with documented but dynamic tables etc.

Reply:
I think this fails packaging since we pass and empty spec